backend/ormconfig.json

```diff
@@ -8,12 +8,12 @@
   "synchronize": true,
   "dropSchema": true,
   "logging": false,
-  "entities": ["src/entity/**/*.ts"],
-  "migrations": ["src/migration/**/*.ts"],
-  "subscribers": ["src/subscriber/**/*.ts"],
+  "entities": ["src/main/entity/**/*.ts"],
```

Comment:
Made folder structure to be like this:
![image](https://user-images.githubusercontent.com/49849754/98451877-ee7ec700-20fe-11eb-827b-39fbf4b3f611.png)